src/main/kotlin/net/maizegenetics/phgv2/cli/RefRangeFasta.kt

```diff
@@ -0,0 +1,113 @@
+package net.maizegenetics.phgv2.cli
+
+import biokotlin.util.bufferedWriter
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.parameters.options.default
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.options.required
+import net.maizegenetics.phgv2.api.HaplotypeGraph
+import net.maizegenetics.phgv2.api.ReferenceRange
+import net.maizegenetics.phgv2.utils.Position
+import net.maizegenetics.phgv2.utils.loadRanges
+import net.maizegenetics.phgv2.utils.seqFromAGC
+import org.apache.logging.log4j.LogManager
+import java.io.File
+
+/**
+ * Write one Fasta file reference range specified by the bedfile.
```

Comment:
Add the word "per" in this sentence to make it clearer (I had trouble understanding at first).  "Write one Fasta file per reference rnage specified by the bedfile"

Reply:
Ok I will add that in the refactored version